preference sidebar incorrect render fix + badge center alignment

MrRobotjs opened the pull request:
# Fixed
- Fixed: Fixed the preference sidebar not rendering correctly at all in Firefox and Chrome.
- Fixed: Fixed the alignment of the badge on the sidebar not being centered with the text
- Fixed Homepage sidebar tabs were not sized properly for when the sidebar was closed

# Chrome
![chrome_QyW5ZhM7lM](https://github.com/user-attachments/assets/49110ead-4bdc-4fdc-a77a-98d8477f4cc3)

# Firefox
![firefox_MCE3fMSFFO](https://github.com/user-attachments/assets/8255de5f-f80e-4443-b69f-ea3c8188e028)

# Sidebar tabs sizing fix
<img width="137" alt="chrome_a2t8neQhAS" src="https://github.com/user-attachments/assets/4534e0bf-61bc-4bd1-b32b-b744594b3b0c" /> <img width="109" alt="MYLr94sj1Q" src="https://github.com/user-attachments/assets/c73bf722-97da-4a8f-831c-3b1be2426f33" />

MrRobotjs committed:
preference sidebar fix + badge center alignment

majora2007 commented:
I'm not understanding what you are fixing? The side nav looks correct for me on Chrome and Firefox.

MrRobotjs commented:
> I'm not understanding what you are fixing? The side nav looks correct for me on Chrome and Firefox.

I have Firefox and Chrome updated to the latest and every time I uncheck the margin-left and check it again then it renders differently. Unless it’s just my PC. I also checked if any extension I had were interfering and it didn’t make a difference.

majora2007 commented:
I just spoke to Diesel and he reported the same, but on mine, I'm not seeing it. I'm a version behind, so I'll update later and validate.

MrRobotjs committed:
fix home sidebar closed state buttons

- was not sized properly for closed state
- different sizing for firefox since firefox doesnt insert the scrollbar like chrome does

MrRobotjs commented:
<img width="137" alt="chrome_a2t8neQhAS" src="https://github.com/user-attachments/assets/4534e0bf-61bc-4bd1-b32b-b744594b3b0c" /><img width="109" alt="MYLr94sj1Q" src="https://github.com/user-attachments/assets/c73bf722-97da-4a8f-831c-3b1be2426f33" />

majora2007 commented:
Was able to reproduce the original issue, thanks for the tweak. I'll merge this into my branch that has other smaller PRs.

majora2007 commented:
@MrRobotjs a user is reporting this on the nightly. Can you please take a look?

> On a tablet in landscape orientation was the inclusion of some text when menu is in what was previously icon only

![image](https://github.com/user-attachments/assets/562b7c9b-bb7b-4eaf-bdd1-0fc11e537936)
